app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -1417,4 +1435,47 @@ public void showMyShotOnBoarding() {
     public Fragment getFragment() {
         return this;
     }
+
+    public void setNightModeEnabled(boolean enable) {
```

Comment:
maybe rename setNightModeEnabled to updateThemeState

Reply:
I think setNightModeEnabled is more appropriate now since we haven't support other theme feature like what night mode does. We can refactor this if necessary in the future.